### PR TITLE
Add COC

### PR DESCRIPTION
**Pull request by indexzero**
Although this is now covered in our [`.github`](https://github.com/godaddy/.github) for the organization I'd prefer to have it here as well.

### CODE REVIEW

File: packages/react-native/.npmignore [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+#
+haul-debug.log
+
+# Ignore generated `storybook/stories/index.js`
+storybook/stories/index.js
+metro.config.js
+
+# The files are added using the `react-native eject` command for development
+ios
+android
```
Comment (msluther) [inline packages/react-native/.npmignore]: Do we need these for the react-native app that we ship in here?

File: packages/react-native/.npmignore [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+### ▲ .gitignore contents above
+### ▼ additional ignore files for publishing
+
+# Coverage directory used by tools like istanbul
+coverage
+.nyc_output
```
Comment (msluther) [inline packages/react-native/.npmignore]: shouldn't those 2 be in `.gitignore` also?

File: packages/react-native/.npmignore (hunk 1 above)
Comment (indexzero) [reply packages/react-native/.npmignore]: Yeah that was my understanding.

File: packages/react-native/.npmignore (hunk 2 above)
Comment (indexzero) [reply packages/react-native/.npmignore]: They are in the top-level `.gitignore` for the entire repository.

**Comment by indexzero** [comment]
Merging since we need this for hygiene and for future publishes.

File: packages/react-native/.npmignore (hunk 1 above)
Comment (indexzero) [reply packages/react-native/.npmignore]: This was addressed by @Swaagie here https://github.com/godaddy/exemplar/pull/11#discussion_r308611300

File: packages/react-native/.npmignore (hunk 2 above)
Comment (indexzero) [reply packages/react-native/.npmignore]: Also addressed in https://github.com/godaddy/exemplar/pull/11#discussion_r311793282

**Pull request merged**